extensions/interactions/MusicNotesInput/directives/MusicNotesInput.js

```diff
@@ -101,8 +102,23 @@ oppia.directive('oppiaInteractiveMusicNotesInput', [
         scope.SOUNDFONT_URL = '/third_party/static/midi-js-2ef687/soundfont/';
         scope.sequenceToGuess = HtmlEscaperService.escapedJsonToObj(
           attrs.sequenceToGuessWithValue);
-        scope.initialSequence = HtmlEscaperService.escapedJsonToObj(
-          attrs.initialSequenceWithValue);
+
+        if (scope.getLastAnswer()) {
+          scope.interactionIsActive = false;
+          scope.initialSequence = scope.getLastAnswer();
+        } else {
+          scope.interactionIsActive = true;
+          scope.initialSequence = HtmlEscaperService.escapedJsonToObj(
+            attrs.initialSequenceWithValue);
+        }
+
+        scope.$on(EVENT_NEW_CARD_AVAILABLE, function( evt, data) {
+          if (data) {
+            scope.interactionIsActive = false;
+            scope.initialSequence = scope.getLastAnswer();
+            scope.reinitStaff();
```

Comment:
Here, I tried putting $(element).draggable('disable') to prevent dragging once answer was submitted and though it worked, it gave console error along with it saying draggable was initialised to element, I tried putting at other places in file and same error came, so I reinitialised the interaction as I am checking for interactionIsActive (and disabling draggable) at all init functions anyways. Is this enough?